### ISSUE TITLE
emacs running in hyper displays file with an underscore under every line

### ISSUE THREAD

**Issue by chetstone** [vercel/hyper#3445]
- [X ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X ] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **MacOS 10.13.6**: 
- **Hyper.app 2.1.1**: 
- **Link of a [Gist] N/A**: 
- **Relevant information from devtools** N/A_: 
- **`Is Vanilla`**: 

## Issue

When running `emacs -nw` (GNU Emacs 25.2.1) in Hyper, when a file is opened, it shows lines underneath every line of the file, as shown in the attached screenshot. When emacs is refreshed with Ctl-l, the lines go away and emacs behaves normally.
![screen shot 2019-01-26 at 22 28 31](https://user-images.githubusercontent.com/976343/51797130-65adca00-21bb-11e9-990e-d462ead791cb.png)

MacOS Terminal doesn't have this issue.

**Comment by chabou** [comment]
I can't reproduce your issue with my emacs (v22.1.1) 🤔 

Can you try to reproduce with a vanilla emacs?

**Comment by chetstone** [comment]
No it doesn't happen with vanilla emacs. I use the popular [Prelude](https://github.com/bbatsov/prelude) customization package and the issue happens with a vanilla Prelude installation. I thought it might be related to the default [Zenburn theme](https://github.com/bbatsov/zenburn-emacs) that comes with Prelude, so I tried disabling that. Depending on how I go about disabling it, the lines may appear for only a couple of seconds when the first file or buffer is loaded, then go away.

**Comment by chabou** [comment]
Thank you for this feedback.

Certainly related to: https://github.com/xtermjs/xterm.js/issues/1147

**Comment by chetstone** [comment]
Thanks for tracking this down. Confirmed by setting TERM=ansi before starting emacs. Lines are gone, but there are other problems: I think emacs really needs xterm. Emacs compatibility could be a show-stopper for me in using hyper. I love the autoprofile plugin but I can't get the option key to work as meta in emacs and my fingers are well-trained. Judging by the number of open issues I see about this it doesn't look like that'll be fixed anytime soon either. Thanks again for the quick response.

**Comment by chabou** [comment]
🙌 Thank you for the workaround (using `ctrl+l` to clear and force a redraw)

I am really happy that `Hyper` suits your needs now.
(And I agree, `hyper-autoprofile` is the best plugin of the universe :trollface:)

Do not hesitate to post any other feedback.

**Comment by LabhanshAgrawal** [comment]
Closing as upstream issue https://github.com/xtermjs/xterm.js/issues/1147 is closed
Feel free to open a new issue if you face any problems
